config/rbac/role.yaml

```diff
@@ -32,6 +32,7 @@ rules:
   - rbac.open-cluster-management.io
   resources:
   - clusterpermissions
+  - multiclusterroleassignments
```

Comment:
Matthew, I think this is a good start. We'll need `managedclustersets` as well and this might need to go in a ClusterRole instead if a role binding doesn't work. ie. create a new ClusterRole and new ClusterRoleBinding and bind to the cluster permission service account.